samples/database.sample.yml

```diff
@@ -0,0 +1,670 @@
+# Mappings to SQLLite Database
+#
+# FST lemma             = lemma.cw
+# FST stem              = fstStem
+# homograph key         = key
+# head                  = lemma.sro
+# inflectional category = senses > sense.category
+
+- fstStem:          kinêpiko-maskotêw
+  generalWordClass: N
+  key:              kinêpiko-maskotêhk
+  lemma:
+    sro: kinêpiko-maskotêhk
```

Comment:
If we consider the FST-lemma the representative word-form of a lexeme, then the lemma for _kinêpiko-maskotêhk_ is _kinêpiko-maskotêw_.

Reply:
Got it. Will correct this.